src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -106,7 +106,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     -->
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(HasRuntimeOutput)' == 'true' and '$(RuntimeIdentifier)' != ''">
```

Comment:
@peterhuene @nguerrera the PR title and description say UseAppHost should be true for fx-dependent apps, but I don't think this actually works for most projects due to the condition `'$(RuntimeIdentifier)' != ''`. @guardrex was trying to write docs for this and it doesn't look like the apphost is produced with the 2.2 SDK by default. https://github.com/aspnet/Docs/issues/9214#issuecomment-441073554.

Was it intentional to require `RuntimeIdentifier` in order to generate the apphost in a fx-dependent app?

Reply:
Intentional for 2.x. See https://github.com/aspnet/Docs/issues/9214#issuecomment-441723840

Reply:
Thanks for clarifying.